lib/pages/chat_details/chat_details.dart

```diff
@@ -66,6 +73,24 @@ class ChatDetailsController extends State<ChatDetails> {
 
   Room? get room => Matrix.of(context).client.getRoomById(roomId!);
 
+  Timeline? _timeline;
+
+  Future<Timeline> getTimeline() async {
+    _timeline ??= await room!.getTimeline();
+    return _timeline!;
+  }
+
+  final mediaListController = SameTypeEventsListController(
+    searchFunc: (event) => event.isVideoOrImage,
+    limit: _mediaFetchLimit,
+  );
+  final linksListController = SameTypeEventsListController(
+    searchFunc: (event) => event.isContainsLink,
+    limit: _linksFetchLimit,
+  );
+
+  final MxcImageCacheMap _mediaCacheMap = {};
```

Comment:
what is it?

Reply:
cache image data into RAM, it won't blink when user scroll